### PR TITLE
Feat (core, ui): Add remaining informed components

### LINKED ISSUE #336: Add remaining Informed components
## User Story
As a developer
I want themed versions of all native form input controls
So I can build consistent forms for my components.

## Acceptance Criteria
AC1. All [native informed Input controls](https://joepuzzo.github.io/informed/?path=/story/inputs--intro) are part of [the context menu form UI specification](https://github.com/johnsonandjohnson/Bodiless-JS/blob/master/packages/bodiless-core/src/Types/ContextMenuTypes.tsx#L36)
AC2. Versions of all controls are provided by bodiless-core-ui and themed against the wireframes.
AC3. There is a page on the test site with the examples of the form elements that matches the wireframes (styled only)

## Resource
Wireframes https://jhdfdc.axshare.com/#id=3ct389&p=aries

### PR DESCRIPTION
**Pull request by konst8**
## Changes
- Add remaining elements from https://joepuzzo.github.io/informed/?path=/story/inputs--intro
- Add example 'form-elements' page

## Test Instructions
- Go to /form-elements page
- Make sure it follows the wireframe https://jhdfdc.axshare.com/#id=3ct389&p=aries

## Related Issues
Closes #336

### CODE REVIEW

File: packages/bodiless-core/src/Types/ContextMenuTypes.tsx [hunk 1]
```diff
@@ -47,13 +47,17 @@ export type UI = {
   ComponentFormUnwrapButton?: ComponentType<HTMLProps<HTMLButtonElement>> | string;
   ComponentFormText?: ComponentType<FieldProps<any, any>>;
   ComponentFormTextArea?: ComponentType<FieldProps<any, any>>;
+  ComponentFormRadioGroup?: ComponentType<FieldProps<any, any>>;
+  ComponentFormRadio?: ComponentType<HTMLProps<HTMLInputElement>>;
+  ComponentFormCheckbox?: ComponentType<FieldProps<any, any>>;
```
Comment (appliedstemcell) [inline packages/bodiless-core/src/Types/ContextMenuTypes.tsx:52]: HTMLProps?

Comment (dewen) [inline packages/bodiless-core/src/Types/ContextMenuTypes.tsx:52]: HTMLProps?

File: packages/bodiless-ui/src/elements.tsx [hunk 2]
```diff
@@ -34,11 +43,13 @@ export const Button = stylable<HTMLProps<HTMLButtonElement>>('button');
 export const Hr = stylable<HTMLProps<HTMLHRElement>>('hr');
 export const Text = stylable<FieldProps<any, any>>(BaseText);
 export const TextArea = stylable<FieldProps<any, any>>(BaseTextArea);
+export const RadioGroup = stylable<FieldProps<any, any>>(BaseRadioGroup);
+export const Radio = stylable<HTMLProps<HTMLInputElement>>(BaseRadio);
+export const CheckBox = stylable<any>(BaseCheckBox);
```
Comment (dewen) [inline packages/bodiless-ui/src/elements.tsx:48]: HTMLProps<HTMLInputElement> ?

**Comment by VancheeZze** [comment]
This looks better, however, the test form now does not have the close and submit buttons, and form elements in the example form are imported directly through `@bodiless/ui` package. I think we want to do it using the `contextMenuForm` from the `@bodiless/core`.

For example:
```js
import { contextMenuForm, getUI } from '@bodiless/core';

const ExampleForm = () => contextMenuForm()(
  ({ ui }: any) => {
    const {
      ComponentFormTitle, ComponentFormFieldWrapper, ComponentFormDescription,
      ComponentFormFieldTitle, ComponentFormText, ComponentFormTextArea,
      ComponentFormRadioGroup, ComponentFormLabel, ComponentFormRadio,
      ComponentFormCheckBox, ComponentFormSelect, ComponentFormOption
    } = getUI(ui);
    return (
      <>
        <ComponentFormTitle>Multi-Field Example Form</ComponentFormTitle>

        <ComponentFormFieldWrapper>
          <ComponentFormDescription>
            Description or instructions.
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </ComponentFormDescription>
        </ComponentFormFieldWrapper>

        // ...
        // Other Demo Form Elements
      </>
    );
  },
);
```

and then

```js
  const getMenuOptions = () => [{
    name: 'Example Form',
    label: 'Test',
    icon: 'article',
    handler: ExampleForm,
  }];
```

Note that this way you don't need to import form elements from the `@bodiless/ui`.

**Pull request merged**